io.catenax.user_estimated_loading/1.0.0/UserEstimatedLoading.ttl

```diff
@@ -0,0 +1,244 @@
+
+:percentage a samm:Property ;
+   samm:preferredName "weightfactor in percantage"@en ;
+   samm:description "Weighfactor of the different profiles, a user has. Different profiles indicates different user as well as different usages of like profiles for daily work routes or for caravan holiday."@en ;
+   samm:characteristic :Percentage ;
+   samm:exampleValue "23.5"^^xsd:decimal .
+
+:trailerUse a samm:Property ;
+   samm:preferredName "traile use switch"@en ;
+   samm:description "Usage of trailer like caravan. "@en ;
+   samm:characteristic :Boolean ;
+   samm:exampleValue true .
+
+:cityDriving a samm:Property ;
+   samm:preferredName "city driving switch"@en ;
+   samm:description "City driving means a short distances."@en ;
+   samm:characteristic :Boolean ;
+   samm:exampleValue true .
+
+:drivingStyle a samm:Property ;
+   samm:preferredName "driving style assessment"@en ;
+   samm:description "numerical value between 1 and 100 describing the driving style. 1 for extreme economic driving, about 30 for normal driving, about 65 for sportive driving, 100 for aggressive driving."@en ;
+   samm:characteristic :WeightFactor ;
+   samm:exampleValue "45.5"^^xsd:decimal .
+
+:hilliness a samm:Property ;
+   samm:preferredName "hilliness assessment"@en ;
+   samm:description "Numerical value between 1 and 100 to describe the hilliness.\n1: absolut flat region, 100 steep mountain roads"@en ;
+   samm:characteristic :WeightFactor ;
+   samm:exampleValue "75.2"^^xsd:decimal .
+
+:curviness a samm:Property ;
+   samm:preferredName "curviness assessment"@en ;
+   samm:description "Numerical value between 1 and 100 describing the curviness (1: straight road ... 100: very curvy road."@en ;
+   samm:characteristic :WeightFactor ;
+   samm:exampleValue "60.0"^^xsd:decimal .
+
+:futureScenario a samm:Property ;
+   samm:preferredName "future scenario"@en ;
+   samm:description "the future scenario cotains one or more future usage profiles where the sum of percentage is 100"@en ;
+   samm:characteristic :FutureScenarioList .
+
+:futureScenarioId a samm:Property ;
+   samm:preferredName "future scenario ID"@en ;
+   samm:description "identification number of future scenarios"@en ;
+   samm:characteristic :PositiveInteger ;
+   samm:exampleValue "1"^^xsd:positiveInteger .
+
+:Mileage a samm-c:Measurement ;
+   samm:preferredName "Mileage @en"@en ;
+   samm:description "distance in kilometre"@en ;
+   samm:dataType xsd:positiveInteger ;
+   samm-c:unit unit:kilometre .
+
+:AgeInMonths a samm-c:Duration ;
+   samm:preferredName "age in months"@en ;
+   samm:dataType xsd:nonNegativeInteger ;
+   samm-c:unit unit:month .
+
+:KbaVariant a samm-c:Enumeration ;
+   samm:preferredName "KBA Variant @en"@en ;
+   samm:description "Current version of the Enumeration is sub-set of list defined from the German Federal Office for motor vehicles. "@en ;
+   samm:dataType xsd:string ;
+   samm-c:values ( "Limousine" "Kombilimousine" "Pick-up" "Van" "SUV" "andere" ) .

```

Comment:
feels a bit odd to have German identifier in a model written in English. However, I assume those are official terms and need to stay like this. Maybe reference the standard or regulation, the come from with a see element